diff --git a/src/joystick/android/SDL_sysjoystick.c b/src/joystick/android/SDL_sysjoystick.c
index cce94b8..c01413f 100644
--- a/src/joystick/android/SDL_sysjoystick.c
+++ b/src/joystick/android/SDL_sysjoystick.c
@@ -572,6 +572,11 @@ SDL_JoystickGUID SDL_SYS_JoystickGetGUID(SDL_Joystick * joystick)
     return guid;
 }
 
+SDL_bool SDL_SYS_IsXInputGamepad_DeviceIndex(int device_index)
+{
+    return SDL_FALSE;
+}
+
 #endif /* SDL_JOYSTICK_ANDROID */
 
 /* vi: set ts=4 sw=4 expandtab: */
diff --git a/src/joystick/bsd/SDL_sysjoystick.c b/src/joystick/bsd/SDL_sysjoystick.c
index 65a32ed..070ef76 100644
--- a/src/joystick/bsd/SDL_sysjoystick.c
+++ b/src/joystick/bsd/SDL_sysjoystick.c
@@ -595,6 +595,11 @@ SDL_JoystickGUID SDL_SYS_JoystickGetGUID(SDL_Joystick * joystick)
     return guid;
 }
 
+SDL_bool SDL_SYS_IsXInputGamepad_DeviceIndex(int device_index)
+{
+    return SDL_FALSE;
+}
+
 static int
 report_alloc(struct report *r, struct report_desc *rd, int repind)
 {
diff --git a/src/joystick/darwin/SDL_sysjoystick.c b/src/joystick/darwin/SDL_sysjoystick.c
index 12c1048..e21458c 100644
--- a/src/joystick/darwin/SDL_sysjoystick.c
+++ b/src/joystick/darwin/SDL_sysjoystick.c
@@ -820,6 +820,11 @@ SDL_JoystickGUID SDL_SYS_JoystickGetGUID(SDL_Joystick *joystick)
     return joystick->hwdata->guid;
 }
 
+SDL_bool SDL_SYS_IsXInputGamepad_DeviceIndex(int device_index)
+{
+    return SDL_FALSE;
+}
+
 #endif /* SDL_JOYSTICK_IOKIT */
 
 /* vi: set ts=4 sw=4 expandtab: */
diff --git a/src/joystick/dummy/SDL_sysjoystick.c b/src/joystick/dummy/SDL_sysjoystick.c
index 9baa795..733525f 100644
--- a/src/joystick/dummy/SDL_sysjoystick.c
+++ b/src/joystick/dummy/SDL_sysjoystick.c
@@ -123,6 +123,11 @@ SDL_JoystickGUID SDL_SYS_JoystickGetGUID(SDL_Joystick * joystick)
     return guid;
 }
 
+SDL_bool SDL_SYS_IsXInputGamepad_DeviceIndex(int device_index)
+{
+    return SDL_FALSE;
+}
+
 #endif /* SDL_JOYSTICK_DUMMY || SDL_JOYSTICK_DISABLED */
 
 /* vi: set ts=4 sw=4 expandtab: */
diff --git a/src/joystick/haiku/SDL_haikujoystick.cc b/src/joystick/haiku/SDL_haikujoystick.cc
index 0caea17..5e3f785 100644
--- a/src/joystick/haiku/SDL_haikujoystick.cc
+++ b/src/joystick/haiku/SDL_haikujoystick.cc
@@ -269,6 +269,11 @@ extern "C"
         return guid;
     }
 
+    SDL_bool SDL_SYS_IsXInputGamepad_DeviceIndex(int device_index)
+    {
+        return SDL_FALSE;
+    }
+
 };                              // extern "C"
 
 #endif /* SDL_JOYSTICK_HAIKU */
diff --git a/src/joystick/iphoneos/SDL_sysjoystick.m b/src/joystick/iphoneos/SDL_sysjoystick.m
index a402759..b501a1f 100644
--- a/src/joystick/iphoneos/SDL_sysjoystick.m
+++ b/src/joystick/iphoneos/SDL_sysjoystick.m
@@ -187,4 +187,9 @@ SDL_JoystickGUID SDL_SYS_JoystickGetGUID(SDL_Joystick * joystick)
     return guid;
 }
 
+SDL_bool SDL_SYS_IsXInputGamepad_DeviceIndex(int device_index)
+{
+    return SDL_FALSE;
+}
+
 /* vi: set ts=4 sw=4 expandtab: */
diff --git a/src/joystick/linux/SDL_sysjoystick.c b/src/joystick/linux/SDL_sysjoystick.c
index 05bf086..e363f9e 100644
--- a/src/joystick/linux/SDL_sysjoystick.c
+++ b/src/joystick/linux/SDL_sysjoystick.c
@@ -880,6 +880,11 @@ SDL_JoystickGUID SDL_SYS_JoystickGetGUID(SDL_Joystick * joystick)
     return joystick->hwdata->guid;
 }
 
+SDL_bool SDL_SYS_IsXInputGamepad_DeviceIndex(int device_index)
+{
+    return SDL_FALSE;
+}
+
 #endif /* SDL_JOYSTICK_LINUX */
 
 /* vi: set ts=4 sw=4 expandtab: */
diff --git a/src/joystick/psp/SDL_sysjoystick.c b/src/joystick/psp/SDL_sysjoystick.c
index 9a2362e..d30cd40 100644
--- a/src/joystick/psp/SDL_sysjoystick.c
+++ b/src/joystick/psp/SDL_sysjoystick.c
@@ -265,5 +265,10 @@ SDL_JoystickGUID SDL_SYS_JoystickGetGUID(SDL_Joystick * joystick)
     return guid;
 }
 
+SDL_bool SDL_SYS_IsXInputGamepad_DeviceIndex(int device_index)
+{
+    return SDL_FALSE;
+}
+
 /* vim: ts=4 sw=4
  */
